recipes/mingw-w64/all/conandata.yml

```diff
@@ -1,18 +1,45 @@
 sources:
   "8.1":
     url:
-      x86_64:
-        posix:
-          seh:
-            url: "http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/seh/x86_64-8.1.0-release-posix-seh-rt_v6-rev0.7z"
-            sha256: "853970527b5de4a55ec8ca4d3fd732c00ae1c69974cc930c82604396d43e79f8"
-          sjlj:
-            url: "http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/sjlj/x86_64-8.1.0-release-posix-sjlj-rt_v6-rev0.7z"
-            sha256: "0e994b33dc92576ba92377e2c18c0f96a04c684fa2d7a3546890819964275a1f"
-        win32:
-          seh:
-            url: "http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-win32/seh/x86_64-8.1.0-release-win32-seh-rt_v6-rev0.7z"
-            sha256: "73797b9a1d0b007dbdd6f010e6064dc0aa4df835b042f6947f73a3bf9cc348c5"
-          sjlj:
-            url: "https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-win32/sjlj/x86_64-8.1.0-release-win32-sjlj-rt_v6-rev0.7z"
-            sha256: "e8c65ddc655534b0330f66f7b480565621e8617cda9937d76ba141a22bf3b2fa"
+      Windows:
+        x86_64:
+          posix:
+            seh:
+              url: "http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/seh/x86_64-8.1.0-release-posix-seh-rt_v6-rev0.7z"
+              sha256: "853970527b5de4a55ec8ca4d3fd732c00ae1c69974cc930c82604396d43e79f8"
+            sjlj:
+              url: "http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-posix/sjlj/x86_64-8.1.0-release-posix-sjlj-rt_v6-rev0.7z"
+              sha256: "0e994b33dc92576ba92377e2c18c0f96a04c684fa2d7a3546890819964275a1f"
+          win32:
+            seh:
+              url: "http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-win32/seh/x86_64-8.1.0-release-win32-seh-rt_v6-rev0.7z"
+              sha256: "73797b9a1d0b007dbdd6f010e6064dc0aa4df835b042f6947f73a3bf9cc348c5"
+            sjlj:
+              url: "https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win64/Personal%20Builds/mingw-builds/8.1.0/threads-win32/sjlj/x86_64-8.1.0-release-win32-sjlj-rt_v6-rev0.7z"
+              sha256: "e8c65ddc655534b0330f66f7b480565621e8617cda9937d76ba141a22bf3b2fa"
+  "8.0":
```

Comment:
I think the 8.0 of the windows build referred to the gcc version.
The most recent version of mingw-w64 is 9.0.

Reply:
I am not 100% sure about this. I checked the versioning of the mingw-w64 ubuntu package, and that is using the mingw release number as the version.

I guess we should also do this here, since you can basically compile mingw 8.0.2 for many different compiler versions. How would you then form the version number of the mingw conan package?

Reply:
Versioning is a hard problem.
I've recently created a recipe for repackaging arm toolchains at https://github.com/conan-io/conan-center-index/pull/6822
There, they just add a date version the toolchains.

I'm currently working on a recipe that builds toolchains for multiple archs/oses.
Because a gnu toolchain entails so much parts (binutis/gcc/glibc/newlib/mingw-w64/...), I'm just going to date version it.

Reply:
Another approach here could also be to add gcc version as an additional option to the mingw package. Since mingw uses proper version number, I would prefer also using these for the conan package itself.

Reply:
That sounds like a nice idea.

Reply:
Ok, added it in an upcoming commit. For now I only add gcc 10.3, but it's easy to extend this if necessary.